lib/main_common.pm

```diff
@@ -1708,6 +1708,7 @@ sub load_extra_tests_console {
     loadtest 'console/osinfo_db' if (is_sle('12-SP3+') && !is_jeos);
     loadtest 'console/libgcrypt' if ((is_sle(">=12-SP4") && (check_var_array('ADDONS', 'sdk') || check_var_array('SCC_ADDONS', 'sdk'))) || is_opensuse);
     loadtest "console/gd";
+    loadtest 'console/valgrind' if (is_sle('>=12-SP4') || is_leap('15.1+') || is_tumbleweed);
```

Comment:
I think it's a good idea in general to treat older versions as exception so you could turn this check around and hence also remove the `is_tumbleweed`, i.e. not include supported versions but exclude unsupported versions. Whenever the yaml schedule is used this code is most likely not evaluated anyway

Reply:
Hi Oliver, thanks for the input! I changed `load_test` accordingly to ensure a more defensive approach, as suggested:

    loadtest 'console/valgrind' if (!is_sle('<=12-SP3'));